packages/isomorphic/src/utils/browser.ts

```diff
@@ -22,6 +22,9 @@ export const bytesToHex: typeof BytesToHexFn = (bytes) => {
 export const hexToBytes: typeof HexToBytesFn = (hex): Uint8Array => {
   const len = hex.length
   const array = new Uint8Array(len / 2)
+  if (!/^[A-F0-9]*$/iu.test(hex)) {
```

Comment:
can this regex be placed in a global variable and re-used?

Reply:
Yes, it could be.